### DISCUSSION TITLE
basic prop, "no overload matches"

### DISCUSSION THREAD

**Discussion by imoldfella** [vuejs/language-tools#662] [Q&A] [answered]
I'm using a basic component
```
export default defineComponent({
  props: {
    title: {
      default: "",
      type: String
    },
    client: Boolean,
  },
```
Then when I try to use the component
`<ChatPane title='xx' />`

I get an error "no overload matches the call"

Sorry if this is trivial, I googled for hours and I'm getting a lot of noise, but nothing that solves the issue. 

It may be relevant that I'm using vue 2 (tons of vuetify, so sadly no option to use vue 3) and may be relevant that I'm using vite. I think I need to do a declare module somewhere, but I haven't figured how yet.

**Comment by johnsoncodehk** [comment, marked as answer]
I have not encountered this problem, if you can provide a minimum reproduction, I was able to check it.
